SoC/gd32vf103/Common/Include/Usb/usb_ch9_std.h

```diff
@@ -98,6 +99,7 @@ enum _usb_desctype {
     USB_DESCTYPE_DEV_QUALIFIER    = 0x6U,        /*!< USB device qualtfier descriptor type */
     USB_DESCTYPE_OTHER_SPD_CONFIG = 0x7U,        /*!< USB other speed configuration descriptor type */
     USB_DESCTYPE_ITF_POWER        = 0x8U,        /*!< USB interface power descriptor type */
+    USB_DESCTYPE_ITF_ASSOCIATION  = 0xbU,        /*!< USB interface association descriptor type */

```

Comment:
Added according to the document [USB Interface Association Descriptor Device Class Code and Use Model](https://usb.org/sites/default/files/iadclasscode_r10.pdf) (Revision 1.0).

Reply:
Is corrected to upper-case 0xBU.